src/pages/explore/ui/pairs.tsx

```diff
@@ -51,8 +54,49 @@ export const ExplorePairs = () => {
 
   const [search, setSearch] = useState('');
 
+  // Fetch trading pair summaries.
   const { data, isLoading, isRefetching, isFetchingNextPage, fetchNextPage } = useSummaries(search);
 
+  // Create temporary translator to normalize volumes/liquidity to USDC values.
+  // Deep clone is neccesary since objects are passed by reference in typeScript.
+  let translator = data?.pages ? structuredClone(data.pages) : undefined;
```

Comment:
issue: I think we should consider moving all of this logic to the backend and have it just return the `Value` type (already denominated in USDC). And then the web app is only responsible for promoting the Value to a ValueView with the registry. Some patterns [here](https://github.com/penumbra-zone/dex-explorer/pull/222) to consider.